src/module/redis_controller.py

```diff
@@ -1,8 +1,48 @@
+from enum import Enum
 import redis
 import logging
 import threading
 import RPi.GPIO as GPIO
 
+class ParameterKey(Enum):
```

Comment:
I have only added parameters for everything that existed before and updated the usage.
I see that there is both REC and IS_RECORDING and maybe they should be the same, but since i don't know the code that well yet i didn't change anything only added what already existed.